packages/preview/curryst/0.4.0/typst.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "curryst"
+version = "0.4.0"
+entrypoint = "curryst.typ"
+authors = ["Rémi Hutin <@remih23>", "Paul Adam <@pauladam94>", "Malo <@MDLC01>"]
+license = "MIT"
+description = "Typeset trees of inference rules."
+repository = "https://github.com/pauladam94/curryst"
+keywords = ["proof tree", "prooftree", "inference", "logic"]
+categories = ["components", "visualization", "integration"]
+disciplines = ["computer-science", "mathematics"]
+compiler = "0.12.0"
+exclude = [".gitignore", "README.md", "examples/*", "tests/*"]
```

It is a good practice to _not_ exclude the README, so that minimal documentation is available offline.

```suggestion
exclude = [".gitignore", "examples/*", "tests/*"]
```